tests/aggregation/tests.py

```diff
@@ -1315,3 +1315,34 @@ def test_aggregation_subquery_annotation_related_field(self):
         # with self.assertNumQueries(1) as ctx:
         #     self.assertSequenceEqual(books_qs, [book])
         # self.assertEqual(ctx[0]['sql'].count('SELECT'), 2)
+
+    def test_ticket26390(self):
+        """
+        Check that the random call does not break GROUP BY clause when
+        used as ordering.
```

Comment:
To add more explanation, docstrings should state the expected behavior and omit prefixes like "Check that" since all tests check things. This guideline is from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).

For example: _"Random ordering is not included in the GROUP BY."_